MySQLdb/times.py

```diff
@@ -18,15 +18,15 @@
 
 def DateFromTicks(ticks):
     """Convert UNIX ticks into a date instance."""
-    return date(*localtime(ticks)[:3])
+    return date(*gmtime(ticks)[:3])
```

Comment:
I understand, just didn't make sense to have these return local timezone. I've reverted the commit.